assets/js/app/Data.js

```diff
@@ -9,7 +9,13 @@ LocusZoom.DataSources = function() {
     this.sources = {};
 };
 
+LocusZoom.KnownDataSources = [];
```

Comment:
I think this pattern of a single object with methods, storage, and initial recipes all included still works, it just wouldn't behave as a singleton per se. The workflow for the implementer would still be to instantiate a new object a la `var mydatasources = new LocusZoom.DataSources();`, and then they can add/modify/remove as they see fit by using methods on the `mydatasources` object. When ready `mydatasources` gets passed into the populate method to be specific to the instance that's then created. By this pattern each LocusZoom instance would retain its own copy of the built-in and whatever customized data sources it was given at the populate step, and that may include slight modifications to built-in sources from instance to instance.